### PR TITLE
Limit Swift line lengths to 100

### PR DESCRIPTION
**Pull request by paulb777**

### CODE REVIEW

File: CoreOnly/Tests/FirebasePodTest/FirebasePodTest/AppDelegate.swift [hunk 1]
```diff
@@ -36,20 +39,31 @@ class StorageExists: Storage {}
...
     return true
   }
 
   // MARK: UISceneSession Lifecycle
 
-  func application(_ application: UIApplication, configurationForConnecting connectingSceneSession: UISceneSession, options: UIScene.ConnectionOptions) -> UISceneConfiguration {
+  func application(
+    _ application: UIApplication,
+    configurationForConnecting connectingSceneSession: UISceneSession,
+    options: UIScene.ConnectionOptions
```
Comment (ryanwilson) [inline CoreOnly/Tests/FirebasePodTest/FirebasePodTest/AppDelegate.swift:55]: Not blocking, but can we see afterwards if we could have it wrap like this:

```
func application(_ application: UIApplication,
                 configurationForConnecting connectingSceneSession: UISceneSession,
                 option: UIScene.ConnectionOptions
) -> UISceneConfiguration
```

Seems a bit more readable having the first argument alongside the function name, I realize that wouldn't work with all functions though.

File: ZipBuilder/Sources/ZipBuilder/ZipBuilder.swift [hunk 2]
```diff
@@ -205,13 +206,18 @@ struct ZipBuilder {
     }
 
     let zipDir = try assembleDistributions(inProjectDir: projectDir, withPackageKind: "Firebase",
```
Comment (ryanwilson) [inline ZipBuilder/Sources/ZipBuilder/ZipBuilder.swift:208]: Hmm should `withPackageKind:` also be wrapped?

File: CoreOnly/Tests/FirebasePodTest/FirebasePodTest/AppDelegate.swift (hunk 1 above)
Comment (paulb777) [reply CoreOnly/Tests/FirebasePodTest/FirebasePodTest/AppDelegate.swift]: Good catch!  Many changes look better after adding the option `--wrapparameters afterfirst`

File: ZipBuilder/Sources/ZipBuilder/ZipBuilder.swift (hunk 2 above)
Comment (paulb777) [reply ZipBuilder/Sources/ZipBuilder/ZipBuilder.swift:208]: Seems like it, but I can't find the right `swiftformat`.  It's stable after manually updating.

**Pull request merged**